packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskHelp/TaskHelp.js

```diff
@@ -56,7 +60,7 @@ function TaskHelp (props) {
           <Box pad={{ top: 'small' }}>
             <Button
               onClick={() => setShowModal(false)}
-              label={counterpart('TaskHelp.close')}
+              // label={counterpart('TaskHelp.close')}
```

Comment:
in theory would be added to packages/lib-classifier/src/translations/en.json or fr.json, similar to `TaskHelp.label`